src/pybind/mgr/cephadm/ssh.py

```diff
@@ -136,7 +136,13 @@ async def _execute_command(self,
                                addr: Optional[str] = None,
                                ) -> Tuple[str, str, int]:
         conn = await self._remote_connection(host, addr)
-        sudo_prefix = "sudo " if self.mgr.ssh_user != 'root' else ""
+        if self.mgr.ssh_user != 'root':
+                if self.mgr.ssh_password:
```

Comment:
nitpick, but we usually just use two spaces for indentation. This section of code being added seems to be using a few more than that.